test/testrunner.py

```diff
@@ -345,13 +345,22 @@ def main():
         # Execute dlc_tests pytest command
         pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]
 
+        is_habana_image = any("habana" in image_uri for image_uri in all_image_list)
         if specific_test_type == "ec2":
+            if is_habana_image:
+                context = Context()
+                context.run("git clone https://github.com/HabanaAI/gaudi-test-suite.git")
```

Comment:
Why is it here? It a prerequisite for habana test only, right? So it should be somewhere in ec2 test fixtures?

Reply:
Tried the `Session` or `autouse` fixtures, they seem to rerun for each function. I will deep dive on the pytest configuration and follow up on this.